src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import AddRemoveForm from '../AddRemoveForm'
+import { shallow } from 'enzyme'
+import mockData from 'mockData'
+
+describe('<AddRemoveForm />', () => {
+  let wrapper
+  const mockProps = {
+    terms: {
+      title: 'Mock Terms',
+      // labelProp: 'name', // default
+      fetchItems: () => mockData.terms.getAll()
+    },
+    questions: {
+      title: 'Mock Questions',
+      labelProp: 'title',
+      fetchItems: () => mockData.questions.getAll()
+    }
+  }
+
+  beforeEach(() => {
+    wrapper = shallow(<AddRemoveForm {...mockProps.terms} />)
+  })
```

Comment:
We're defaulting the shallow render to use Terms. The only module that is different right now is the Questions, which uses a different `labelProp`, so we test that separately later on.